src/imitation/rewards/reward_nets.py

```diff
@@ -169,6 +169,7 @@ def predict_processed(
         action: np.ndarray,
         next_state: np.ndarray,
         done: np.ndarray,
+        **kwargs,
```

Comment:
Consider adding `del kwargs` to function body to self-document that these keyword arguments are ignored by this function.

Reply:
I've added this in a couple of places see 0246fb2713f0d70b3eed81132ae711d0e9940cc1.